src/spatial/collision.js

```diff
@@ -483,35 +462,35 @@ Crafty.c("Collision", {
      * ~~~
      */
     ignoreHits: function () {
-      var components = arguments;
-      var i = 0;
-      var collisionData;
+        var components = arguments;
```

Comment:
I made a couple of code align changes (from 2 to 4 spaces), if It's not suits you - please tell me, I'll revert it back.